packages/dapp/src/pages/profile/[id]/index.tsx

```diff
@@ -118,44 +113,46 @@ const Home = () => {
               <HStack width="100%">
                 <Avatar
                   src={
-                    basicProfile.image
+                    basicProfile?.image
                       ? IPFS_GATEWAY +
-                        basicProfile.image.original.src.split("//")[1]
+                        basicProfile?.image.original.src.split("//")[1]
                       : "https://source.unsplash.com/random"
                   }
                   size="2xl"
                 />
                 <VStack paddingX={4} paddingY={6} align="start">
                   <Text fontSize="4xl">
-                    {basicProfile.emoji}{" "}
-                    {basicProfile.name ? basicProfile.name : "Anonymous"}
+                    {basicProfile?.emoji}{" "}
+                    {basicProfile?.name ? basicProfile?.name : "Anonymous"}
                   </Text>
                   <Text fontSize="lg" color="purple.200">
-                    {basicProfile.description}
+                    {basicProfile?.description}
                   </Text>
                   <Text fontSize="lg" color="purple.200">
-                    <Icon as={FaMapMarkerAlt} /> {basicProfile.homeLocation},{" "}
-                    {basicProfile.residenceCountry}
+                    <Icon as={FaMapMarkerAlt} /> {basicProfile?.homeLocation},{" "}
+                    {basicProfile?.residenceCountry}
                   </Text>
                 </VStack>
               </HStack>
               <Spacer mt={4} />
               <Heading fontSize="2xl" color="purple.200" pt={5}>
                 Skills
               </Heading>
-              {publicProfile && publicProfile.skillTags ? (
+              {publicProfile && publicProfile?.skillTags ? (
                 <SimpleGrid columns={6} spacing={4}>
-                  {publicProfile.skillTags.map((skill, index) => (
-                    <Tag
-                      bgColor="purple.600"
-                      textAlign="center"
-                      size="lg"
-                      key={index}
-                      minWidth="fit-content"
-                    >
-                      {skill}
-                    </Tag>
-                  ))}
+                  {publicProfile?.skillTags.map(
```

Comment:
I think we should check for publicProfile?.skillTags too.

Reply:
> I think we should check for publicProfile?.skillTags too.

Can `skillTags` be `undefined`? If it's an empty array, map will account for it

Reply:
I got an error from one profile the other day because skillTags was null. We can add this check to avoid an error.